daemon/networkdriver/bridge/driver.go

```diff
@@ -379,6 +381,13 @@ func Allocate(job *engine.Job) engine.Status {
 		return job.Error(err)
 	}
 
+	// If no explicit gateway was given, use default.
+	if requestedGateway != nil {
```

Comment:
I will change my code and add some documentation. Anyway, thanks for code review :)